packages/commerce-sdk-react/mock-responses/usecategories-returns-data.json

```diff
@@ -6,21 +6,21 @@
-            "a106e8f11ebafe80d58fd079ab3591fa841fa1872b16451c888f52054ec1699b0ba2828103abf1f3e6305eec346e160cda9e19f493a8cd4edac6415b7ccee688d9ec940d11b2f5bb0ec7f481886d033586038e9bb38dcb087831253bc1c81e8df40162b37e3a793b5d28465a2a9b60822fe5a6ab3f31d3995f4dbf8a694b4b0748e9caa3a4c169c4e8e87765b45f455708111daca11f000000ffffa2fa343417cc24b0294a25f9258939a0edd25cb500000000ffff0300772c72733f3d0000"
+            "1f8b0800000000000003ec544d6fd4400cbdf7575839944b3efaa122d86b7aa9a0ea612b7140a8f2ce38c9c064bc1a3bdd6e2bfe3b4aba5b9afd28502e1cb84419fbc5e3f75eec870380c4bbd669328193b43f59544c26f0f90000e06178022406956a8e8ee429f73c3f609c4d26902cb8a52019775a3995241d215aaca90735aa739914c5fd7dacb2a3a3e35c30d819dfe59d1c9de6f62e37dcb6140d19cf9dcd053d49c5d1509f283814965a0c768191725154678aac983a25c94439521539686650d1739db559e09051ff55859dd7c22ede99b3f727b3b75531b4248578d642bad9cd0c43a0288541cd1edfb331a3fcebbc1eb30ad80ea4ae76519e634de72426bab93a0e3d8ef7e1ae9d7a7a01112968f968c4f2e2b9d8c6b3362ed42f7d701d8946de6dfab7e5e1a8da88e9a7c7fc28fd3dfdfdcabb1b1e5d51ae11afbf64978a3b0d1b5ff1ecf465a4a7b9a180334f25b7bdb4c9042af4421b18697871112e2974c904347674b0a3f9fd73b35626539effb3d3737c7a466748f60fa6a7a7b37f74aeb7c8ee989b69c373187ebd3702d3c645951ca60d47050c163c871ac413dd520a577715479b82b418350583d2a11f60c6a3883320baf4245071842ecc08b5b7160cb7555fb08adcc292bb0896e41b28833604752454e04e2d7314408572a5349483d4db1c3ed072c1d1ca4f8b41fad653589d6c24d98c2df896c2468cabca19da08ae780dc117f6c95ab45e6670c1f8ceba50afb43b848fbd72d39572e550328572a5d3215c72a4ffabe8af56d1fea9de1e83576fa37ed9fc6a196de6dd3d950d46bd384f7e000000ffff221487c9f1a06cee04ced59e83a62832484933b534354e45298a7212f340095c1f5be8830b2132cb64705e4d1db4c5724aaa8985b161622a09c532d447b84b66176c5e265438434d0517ae8939390af9c9c989c599f979c8854f5a7e512e",
```

Comment:
so moving forward we are going to have many mock files like this and it's gonna be a pain to review any PR for this package because when we make changes, we update all/many of the mock files.

I'm wondering if there is a way to configure GitHub to don't show the diffs in certain folder.

Reply:
seems like this is a common problem and there is a solution: https://stackoverflow.com/a/58923410/5666628

Reply:
Marked mock-responses folder as generated so the files are still tracked for changes but won't show-up in PR diffs

Reply:
Oh nice that we can do that

Reply:
For some reason it only worked for a bunch of files :/